recipe/run_test.sh

```diff
@@ -1,8 +1,5 @@
 #!/bin/bash -e
 
-# Test the ipopt binary
-ipopt mytoy.nl | grep -q "Optimal Solution"
-
```

Comment:
are you sure this involves asl ? i dont read asl anywhere in this file

Reply:
I don't actually know what ASL does, but I thought that the nl files were AMPL problem description files and that AMPL and ASL were somehow the same thing. Or maybe AMPL provides a nl file reader and an ASL solver. I thought this line failed without ASL in my second commit.

Reply:
I can try again.

Reply:
seems you're right: ipopt exe is not built without asl
I'd like to leave asl enabled then, maybe moving it to a separate package would be ok for you ?

Reply:
Moving it to a separate feedstock would be ideal, so that the licensing is properly addressed and things aren't vendored. But maybe it is sufficient to use: https://github.com/conda-forge/ampl-mp-feedstock